src/resources/MachineLearning/MachineLearningInterfaces.ts

```diff
@@ -23,6 +23,15 @@ export interface CommerceSupport {
     [key: string]: any;
 }
 
+export interface IndexExport {
+    query: string;
+}
+
+export interface DocumentContentSource {
+    objectType: string;
+    sourceLocations: [string];
```

Comment:
Is it an array of a single string or a string array? With the current typing it will only have one value


```suggestion
    sourceLocations: string[];
```